src/styles/main.css

```diff
@@ -10,9 +10,18 @@ html {
 }
 
 * {
-  -webkit-transition: border 200ms linear, background-color 200ms linear;
-  -moz-transition: border 200ms linear, background-color 200ms linear;
-  -o-transition: border 200ms linear, background-color 200ms linear;
-  -ms-transition: border 200ms linear, background-color 200ms linear;
   transition: border 200ms linear, background-color 200ms linear;
+  transition: border 200ms linear, background-color 200ms linear;
+  transition: border 200ms linear, background-color 200ms linear;
+  transition: border 200ms linear, background-color 200ms linear;
+  transition: border 200ms linear, background-color 200ms linear;
+}
+
+.logotype-big {
+  max-height: 40px;
```

Comment:
I see how this could be nice for a validator like Staking Fund used in the screenshot, where they have a wide logo image. We can add a max-width here too if there's a problem with images that are too wide later.

Reply:
Yeah, there appear to be some validators with non-rectangular logos like this one
![image](https://user-images.githubusercontent.com/83672/144064104-e7fc390b-aa27-489c-9819-008858fae946.png)

Reply:
I added `max-width` for `.logotype-big` too, so that no one can break the layout by having malicious logotype size.